File: Problem-9/OddEven_bitwise.c
// WAP to find whether a number is Odd or Even without using a % operator.

// Using bitwise AND

#include<stdio.h>

void main(){
    int n;

    printf("Enter n : ");
    scanf("%d", &n);

    if((n & 1) == 0){
        printf("%d is Even !!", n);
    } 
    else{
        printf("%d is Odd !!", n);
    }
}

File: Problem-9/OddEven_bitwise.java
// WAP to find whether a number is Odd or Even without using a % operator.

// Using bitwise AND

import java.util.*;

public class OddEven_bitwise{
    public static void main(String [] args){

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter n : ");
        int n = sc.nextInt();

        if((n & 1) == 0){
            System.out.println(n + " is Even !!");
        }
        else{
            System.out.println(n + " is Odd !!");
        }
    }
}